packages/inter-protocol/src/feeDistributor.js

```diff
@@ -289,10 +290,16 @@ export const makeFeeDistributor = (feeIssuer, terms) => {
      */
     setDestinations: async newDestinations => {
       destinations = newDestinations;
-      shareConfig = makeShareConfig(destinations, terms.keywordShares);
+      shareConfig = makeShareConfig(destinations, keywordShares);
       // Run once immediately for these destinations.
       await schedulePayments();
     },
+
+    /** @param {Record<Keyword, bigint>} newShares */
+    setKeywordShares: newShares => {
```

Comment:
adding this to the creatorFacet means that it's accessible only to the Builder DAO, right?

Reply:
yes; that's the goal of #6926